folder_header.html

```diff
@@ -35,19 +35,20 @@ <h1 class="d-lg-block d-none" id="title-text">GRAPHIC AND MULTIMEDIA DESIGN</h1>
                         <li><a class="dropdown-item" href="../suggested_books/history.html">HISTORY</a></li>
                         <li><a class="dropdown-item" href="../suggested_books/monographs.html">MONOGRAPHS</a></li>
                         <li><a class="dropdown-item" href="../suggested_books/design_basics.html">DESIGN BASICS</a>
+                        <li><a class="dropdown-item" href="../suggested_books/design_basics.html">COLOR THEORY</a>
```

Comment:
Is this new line suppose to have `../suggested_books/color_theory.html` as an href? 

@sseci